network/wsNetwork.go

```diff
@@ -1110,21 +1148,23 @@ func (wn *WebsocketNetwork) ServeHTTP(response http.ResponseWriter, request *htt
 
 	client, _ := wn.GetHTTPClient(trackedRequest.remoteAddress())
 	peer := &wsPeer{
-		wsPeerCore:            makePeerCore(wn.ctx, wn, wn.log, wn.handler.readBuffer, trackedRequest.remoteAddress(), client, trackedRequest.remoteHost),
-		conn:                  wsPeerWebsocketConnImpl{conn},
-		outgoing:              false,
-		InstanceName:          trackedRequest.otherInstanceName,
-		incomingMsgFilter:     wn.incomingMsgFilter,
-		prioChallenge:         challenge,
-		createTime:            trackedRequest.created,
-		version:               matchingVersion,
-		identity:              peerID,
-		identityChallenge:     peerIDChallenge,
-		identityVerified:      atomic.Uint32{},
-		features:              decodePeerFeatures(matchingVersion, request.Header.Get(PeerFeaturesHeader)),
-		enableVoteCompression: wn.config.EnableVoteCompression,
+		wsPeerCore:                      makePeerCore(wn.ctx, wn, wn.log, wn.handler.readBuffer, trackedRequest.remoteAddress(), client, trackedRequest.remoteHost),
+		conn:                            wsPeerWebsocketConnImpl{conn},
+		outgoing:                        false,
+		InstanceName:                    trackedRequest.otherInstanceName,
+		incomingMsgFilter:               wn.incomingMsgFilter,
+		prioChallenge:                   challenge,
+		createTime:                      trackedRequest.created,
+		version:                         matchingVersion,
+		identity:                        peerID,
+		identityChallenge:               peerIDChallenge,
+		identityVerified:                atomic.Uint32{},
+		features:                        decodePeerFeatures(matchingVersion, request.Header.Get(PeerFeaturesHeader)),
+		enableVoteCompression:           wn.config.EnableVoteCompression,
+		voteCompressionDynamicTableSize: wn.voteCompressionDynamicTableSize,
```

Comment:
Could we throw in some blank lines for grouping here (and in the struct definition) so we're not doomed to big diffs the next time we add a long name again?